frontend/source/scss/components/atoms/_headline.scss

```diff
@@ -19,17 +19,14 @@ New modifiers can be added below.
 ### MODIFIERS:
 
 h1 - h6
-    Self explaining.
+  All values in em depending on base font-size of 16 px.
```

Comment:
this is a little picky, but when em isn't specified the default is based on 16px. Perhaps this explanation isn't needed.